rholang/src/main/scala/rholang/rosette/Roselang.scala

```diff
@@ -497,7 +497,7 @@ extends AllVisitor[VisitorTypes.R,VisitorTypes.A] {
                   B("")(procTerm, pTerm) // TODO: Potentially allow StrTermPtdCtxtBr without Namespace ?
                 }
 
-                val matchTerm = B(_match)(TS, qPattern, pTerm)
+                val matchTerm = B(_match)(qPattern, pTerm)
```

Comment:
Why is this here?

I added this on purpose.

Your commit message does not give enough information as to why you think it's safe or appropriate to undo this change.

Reply:
I changed the "match_pattern" function to not be dependent on the namespace instance.

Reply:
I'll allow it, but in the future, that seems like it should go in a separate PR